src/electionguard_cli/e2e_steps/election_record_step.py

```diff
@@ -0,0 +1,52 @@
+from shutil import make_archive
+from os.path import splitext
+import tempfile
+import click
+from electionguard_cli.cli_models import BuildElectionResults, E2eSubmitResults
+from electionguard_cli.cli_models.e2e_decrypt_results import E2eDecryptResults
+from electionguard_cli.cli_models.e2e_inputs import E2eInputs
+from electionguard_cli.e2e_steps.e2e_step_base import E2eStepBase
+from electionguard.constants import get_constants
+
+from electionguard_tools.helpers.export import export_record
+
+
+class ElectionRecordStep(E2eStepBase):
+    """Responsible for publishing an election record after an election has completed."""
+
+    REMOVE_RAW_OUTPUT = True
+    REMOVE_ZIP_OUTPUT = True
+    COMPRESSION_FORMAT = "zip"
+
+    def run(
+        self,
+        election_inputs: E2eInputs,
+        build_election_results: BuildElectionResults,
+        submit_results: E2eSubmitResults,
+        decrypt_results: E2eDecryptResults,
+    ) -> None:
+
+        self.print_header("Election Record")
+        guardian_records = [
```

Comment:
good use of comprehension

Reply:
Thanks, trying to be pydantic, but for the record I still think a map/reduce style like `guardian_records = election_inputs.guardians.map(g => g.publish())` is clearer.